setup/setup_validation_docker.sh

```diff
@@ -0,0 +1,239 @@
+
+    # CPU model
+    CPU=`cat /proc/cpuinfo |grep "model name" |awk -F: '{print $2}'| uniq`
+    printf "CPU model              :%-10s $CPU\n"
+
+    # Number of CPU
+    NCPU=`cat /proc/cpuinfo |grep "model name" |awk -F: '{print $2}'| wc -l`
+    if (( $NCPU < 6)); then
+	export OVER="BAD"
+    else
+	export OVER="OK"
+    fi
+    printf "NB CPU                 : %-10s %-10d\n" $OVER $NCPU
+
+    # Memory check
+    MEM=`cat /proc/meminfo |grep "MemTotal:"|awk '{print $2}'`
+    TMEM=`echo "scale = 3; $MEM/1024/1024" |bc`
+
+    if (( $MEM < 67108864 ));then
+	export OVER="BAD"
+    else
+        export OVER="OK"
+    fi
+    printf "RAM                    : %-10s %-10sKB\n" $OVER $MEM
+
+    # SELinux check
+    export OVER="OK"
+    if [ ! \( `getenforce` == "Permissive" -o `getenforce` == "Disabled" \) ]; then
+      export OVER="BAD"
+    fi
+    printf "SE Linux               : %-10s %-10s\n" $OVER `getenforce`
+
+    echo -e "\nDOCKER"
+    echo "======"
+    # Docker version check
+    export OVER="BAD"
+    DOV=`docker -v`
+    if [ -n "$DOV" ];then
+	DOCK=`docker -v|awk '{print $3}'|awk -F. '{print $1}'`
+	if (( $DOCK < 17 ));then
+    	    export OVER="BAD"
+	else
+    	    export OVER="OK"
+	fi
+    else
+	echo "Docker is not installed. Please install." `basename "$0"` "--help"
+    fi
+    printf "Docker installed       : %-10s $DOV\n" $OVER
+
+    # Docker compose version check
+    export OVER="BAD"
+    DCOV=`docker-compose -v`
+    if [ -n "$DCOV" ];then
+	DOCKC=`docker-compose -v|awk '{print $3}'|awk -F. '{print $1" "$2}'`
+	V=($DOCKC)
+	if (( ${V[0]} < 1 && ${V[1]} < 13 ));then
+	    export OVER="BAD"
+	else
+    	    export OVER="OK"
+	fi
+    else
+	echo "Docker-compose is not installed. Please install." `basename "$0"` "--help"
+    fi
+    printf "Docker-compose         : %-10s $DCOV\n" $OVER
+
+    # Docker status
+    export OVER="BAD"
+    DRUN=`rpm -qa|grep docker`
+    if [ -n "$DRUN" ]; then
+    STATUS=`systemctl status docker.service|grep active|awk '{print $2}'`
+	if [ $STATUS = 'active' ];then
+	    export OVER="OK"
+	else
+	    export OVER="BAD"
+	fi
+    else
+	echo "Docker is not installed. Please install." `basename "$0"` "--help"
+    fi
+    printf "Docker-status          : %-10s %-10s\n" $OVER $STATUS
+
+
+
+    # ADDITIONAL INFO
+    echo -e "\nADDITIONAL INFO"
+    echo "==============="
+    s3_hosts=(sg-media-usor-01 sg-media-tokyo sg-media-ireland)
+    for i in ${s3_hosts[@]}; do
+	if ( curl -sS  https://${i}.s3.amazonaws.com|grep "AccessDenied">/dev/null );then
+	    export OVER="OK"
+	else
+	    export OVER="NO"
+	    break
+	fi
+    done
+    printf "S3 available           : %-10s\n" $OVER
+
+
+    # DATABASE INFO
+    echo -e "\nDATABASE INFO"
+    echo "==============="
+    export OVER="NO"
+    PGCHECK=`rpm -qa|grep postgres`
+    if [ -n "$PGCHECK" ];then
+    STATUS=`systemctl status postgresql.service|grep active|awk '{print $2}'`
```

Comment:
Hm... Weird, default postgres comes with centos repo does not contain version number in /var/lib/pgsql/ and "sudo systemctl status postgresql-9.3.service".

Ok, all good then.